CHANGELOG.md

```diff
@@ -74,8 +76,19 @@ attached to all deprecated names to discourage their use.
 * In `Data.Nat.Properties`:
   ```agda
   +-*-suc ↦ *-suc
+
+  ```
+
+* In `Data.Bin.Properties`:
+  ```agda
+  2[1+-≢0  ↦  2[1+x]≢0
+  1+[2-≢0  ↦  1+2[1+x]≢0
```

Comment:
These don't need to be listed as renamings as the originals have never been released.

Reply:
Removed.